lib/Dialect/FIRRTL/Transforms/LowerLayers.cpp

```diff
@@ -649,24 +692,30 @@ LogicalResult LowerLayersPass::runOnModuleBody(FModuleOp moduleOp,
     // parent layer.
     builder.setInsertionPointAfter(layerBlock);
     auto instanceName = instanceNameForLayer(layerBlock.getLayerName());
+    auto innerSym =
+        hw::InnerSymAttr::get(builder.getStringAttr(ns.newName(instanceName)));
+
     auto instanceOp = builder.create<InstanceOp>(
         layerBlock.getLoc(), /*moduleName=*/newModule,
         /*name=*/
-        instanceName, NameKindEnum::DroppableName,
+        instanceName, NameKindEnum::InterestingName,
         /*annotations=*/ArrayRef<Attribute>{},
-        /*portAnnotations=*/ArrayRef<Attribute>{}, /*lowerToBind=*/true,
-        /*doNotPrint=*/false,
-        /*innerSym=*/
-        (innerSyms.empty() ? hw::InnerSymAttr{}
-                           : hw::InnerSymAttr::get(builder.getStringAttr(
-                                 ns.newName(instanceName)))));
-
-    auto outputFile =
-        outputFileForLayer(circuitName, layerBlock.getLayerName());
+        /*portAnnotations=*/ArrayRef<Attribute>{}, /*lowerToBind=*/false,
+        /*doNotPrint=*/true, innerSym);
```

Comment:
I'm still a little hazy on the story around `lowerToBind` vs. `doNotPrint`. This works because `lowerToBind` is replaced with bind ops in the bind files, correct?

Regardless, having both of these is not ideal and would be good to clean up.

Reply:
Yep, that's right.